internal/cli/config/delete.go

```diff
@@ -80,22 +83,27 @@ func DeleteCmd(h *internal.Helper) *cobra.Command {
 				return errors.Trace(err)
 			}
 
-			err = t.Delete(profileName)
+			err = t.Delete(curProfileName)
 			if err != nil {
 				return errors.Trace(err)
 			}
 
 			// If the deleting profile is the current profile, set the current profile to another profile
 			curP := t.Get(prop.CurProfile)
-			if curP == profileName {
+			curPString, ok := curP.(string)
```

Comment:
why this? except `nil`, what could it be?

Reply:
curP is interface{}, we need type conversion here because strings.EqualFold(curPString, curProfileName) need string type.
Previously we use "==" which has implicit type conversion